content/sensu-core/2.0/reference/events.md

```diff
@@ -1,65 +1,192 @@
+
+## Events specification
+
+### Attributes
+timestamp    | 
+-------------|------ 
+description  | The time of the Event occurrence in eopoch time. 
+type         | integer 
+example      | {{< highlight shell >}}"timestamp": 1522099512{{</ highlight >}}
+
+silenced     | 
+-------------|------ 
+description  | If the Event is to be silenced. 
+type         | boolean 
+example      | {{< highlight shell >}}"silenced": false{{</ highlight >}}
+
+check        | 
+-------------|------ 
+description  | The [Check attributes][1] used to obtain the check result. 
+type         | Check 
+example      | {{< highlight json >}}
+  "check": {
+    "check_hooks": null,
+    "command": "check-http-response-time.rb -a example.com -C 5000 -w 3000",
+    "duration": 1.903135228,
+    "environment": "default",
+    "executed": 1522100915,
+    "handlers": [],
+    "high_flap_threshold": 0,
+    "history": [
+      {
+        "executed": 1522100315
+      },
+      {
+        "executed": 1522100915
+      }
+    ],
+    "interval": 300,
+    "last_ok": 1522100916,
+    "low_flap_threshold": 0,
+    "name": "example-check",
+    "occurrences": 1,
+    "occurrences_watermark": 1,
+    "organization": "default",
+    "output": "CheckHttpResponseTime OK: 1635 is acceptable\n",
+    "proxy_entity_id": "",
+    "publish": true,
+    "round_robin": false,
+    "runtime_assets": [],
+    "state": "passing",
+    "status": 0,
+    "stdin": false,
+    "subdue": null,
+    "subscriptions": [
+      "web"
+    ],
+    "timeout": 30,
+    "total_state_change": 0
+  }
+{{</ highlight >}}
+
+entity       | 
+-------------|------ 
+description  | The [entity attributes][2] from the originating entity (agent or proxy). 
+type         | Entity 
+example      | {{< highlight json >}}
+  "entity": {
+    "class": "agent",
+    "deregister": false,
+    "deregistration": {},
+    "environment": "default",
+    "id": "example-agent",
+    "keepalive_timeout": 120,
+    "organization": "default",
+    "redact": [
+      "password",
+      "passwd",
+      "pass",
+      "api_key",
+      "api_token",
+      "access_key",
+      "secret_key",
+      "private_key",
+      "secret"
+    ],
+    "subscriptions": [
+      "web",
+      "entity:example-entity"
+    ],
+    "system": {
+      "hostname": "example",
+      "os": "linux",
+      "platform": "ubuntu",
+      "platform_family": "debian",
+      "platform_version": "16.04",
+      "network": {
+        "interfaces": [
+          {
+            "name": "lo",
+            "addresses": [
+              "127.0.0.1/8",
+              "::1/128"
+            ]
+          },
+          {
+            "name": "eth0",
+            "mac": "52:54:00:20:1b:3c",
+            "addresses": [
+              "93.184.216.34/24",
+              "2606:2800:220:1:248:1893:25c8:1946/10"
+            ]
+          }
+        ]
+      },
+      "arch": "amd64"
+    },
+    "user": "agent"
+  }
+{{</ highlight >}}
+
+## Examples
```

Comment:
I feel like these examples don't belong here, since they use `sensuctl`, and instead we should simply put a full event in JSON here. That being said, maybe we could create a guide around events? But I'm not sure what it could consist of, except maybe resolving events?

Reply:
OK, I'll convert these to JSON. I don't think we need a full guide for events as I think the only thing of note is resolving/viewing events, and that's pretty easy to figure out from the cli help text.